style/roads.mss

```diff
@@ -2615,6 +2628,16 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
         dashes/line-width: 3;
         dashes/line-color: #999;
         dashes/line-dasharray: 1,10;
+        dashes/line-join: round;
+
+        [preserved = 'yes'] {
+          line/line-width: 3;
```

Comment:
prefixes used here differ from those of the other railway types (line/dashes vs. dark/light).

Reply:
I kept this the same as the block of the railway feature styles that this preserved-ness has impact on.

There is indeed a mismatch in the naming of the prefixes, but I think I should not change all that in this PR. It seems out of scope.